site/src/pages/SetupPage/SetupPage.tsx

```diff
@@ -46,10 +31,15 @@ export const SetupPage: FC = () => {
         <title>{pageTitle("Set up your account")}</title>
       </Helmet>
       <SetupPageView
-        isLoading={setupState.hasTag("loading")}
-        error={error}
-        onSubmit={(firstUser) => {
-          setupSend({ type: "CREATE_FIRST_USER", firstUser });
+        isLoading={createFirstUserMutation.isLoading}
+        error={createFirstUserMutation.error}
+        onSubmit={async (firstUser) => {
+          await createFirstUserMutation.mutateAsync(firstUser);
```

Comment:
Forgot to ask earlier, but how has Coder generally approached error handling before? I checked the `App.tsx` file and didn't see a global error handler for the query client/cache, so I wasn't sure where/how we generally try to address errors

Reply:
Assuming error handling is taken care of, this looks good to me!